apps/src/pcd_video_player/pcd_video_player.cpp

```diff
@@ -47,6 +47,7 @@
 #include <QMutexLocker>
 #include <QObject>
 #include <QRadioButton>
+#include <QVTKOpenGLWindow.h>
```

Comment:
The earliest VTK version where this header is available is 8.2.0. Can you add a preprocessor check?

Reply:
I've added a check, but judging from the CI fails, i'm not sure it's the only issue.

Reply:
well at least CI is happy. seems like its not building with current vtk, or at least not all jobs

Reply:
> seems like its not building with current vtk, or at least not all jobs

Yes, we are testing with multiple VTK versions.
The VTK version check doesn't look right. E.g. VTK 9.1.0 returns false here. Which VTK version do you use?
You could just use `VTK_MAJOR_VERSION >= 9` here, neglecting VTK 8.2.0, which would be okay from my point of view. Or you use something more complex to also support VTK 8.2.0.
And let's also use `HAS_QVTKOPENGLWINDOW_H` like in the other PR?

Reply:
indeed it doesnt work on my system also, not sure why